File: Be/property-service/src/main/java/com/property/dto/response/InvitationResponse.java
package com.property.dto.response;

import java.math.BigDecimal;
import java.time.Instant;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class InvitationResponse {

    RoomResponse room;
    RoomTypeResponse roomType;

    String landlordId;

    String invitationStatus;

    String inviteToken;

    String message;
    String createdAt;

    Instant endDate;
    Instant startDate;

    BigDecimal price;
    BigDecimal depositAmount;
}

File: Fe/client/src/dto/response/InvitationResponse.ts
import { roomResponse } from "@/dto/response/roomResponse";
import { roomTypeResponse } from "@/dto/response/roomTypeResponse";

export type InvitationResponse = {
	room: roomResponse;
	roomType: roomTypeResponse;

	landlordId: string;
	invitationStatus: string;
	inviteToken: string;
	message: string;
	createdAt: string;
	price: number;
	depositAmount: number;
	startDate: string;
	endDate: string;
}